modules/generic/src/main/scala/pureconfig/generic/EnumCoproductHint.scala

```diff
@@ -28,7 +28,10 @@ class EnumCoproductHint[A] extends CoproductHint[A] {
     cursor.asString.right.flatMap { str =>
       options.find(str == fieldValue(_)) match {
         case Some(opt) => Right(Use(cursor, opt))
-        case None => cursor.failed[CoproductHint.Action](NoValidCoproductOptionFound(cursor.value, Seq.empty))
+        case None =>
```

Comment:
I'm OK with it either way. If @jcazevedo doesn't object I'll revert this file

Might it be time to remove this deprecated class anyway? It's been 10 months since the commit which marked that type as deprecated? That should certainly be done as a separate PR.